frontend_app_file/src/container/File.jsx

```diff
@@ -642,36 +642,48 @@ class File extends React.Component {
     shareEmailList = shareEmailList.filter(shareEmail => !invalidEmails.includes(shareEmail))
 
     if (invalidEmails.length > 0 || shareEmailList === 0) {
-      this.sendGlobalFlashMessage(props.t(`Error: ${invalidEmails} are not valid`))
-    } else {
-      const fetchResultPostShareLinks = await handleFetchResult(await postShareLinksList(
-        state.config.apiUrl,
-        state.content.workspace_id,
-        state.content.content_id,
-        shareEmailList,
-        state.sharePassword !== '' ? state.sharePassword : null
-      ))
-
-      switch (fetchResultPostShareLinks.apiResponse.status) {
-        case 200:
-          this.setState(prev => ({
-            shareLinkList: [...prev.shareLinkList, ...fetchResultPostShareLinks.body],
-            shareEmails: '',
-            sharePassword: ''
-          }))
-          GLOBAL_dispatchEvent({ type: CUSTOM_EVENT.REFRESH_CONTENT_LIST, data: {} })
-          break
-        case 400:
-          switch (fetchResultPostShareLinks.body.code) {
-            case 2001:
-              this.sendGlobalFlashMessage(props.t('The password length must be between 6 and 512 characters and the email(s) must be valid'))
-              break
-            default: this.sendGlobalFlashMessage(props.t('Error while creating new share link'))
-          }
-          break
-        default: this.sendGlobalFlashMessage(props.t('Error while creating new share link'))
-      }
+      this.sendGlobalFlashMessage(`${props.t('The following emails are not valid:')}
```

Comment:
no need for a line break here.
If you want a line break in the output, use html or \n